keyboards/handwired/leftynumpad/readme.md

```diff
@@ -0,0 +1,14 @@
+# LeftyNumpad
+
```

Comment:
If you have a picture/render of the board, this would be a fine place to showcase it.
See other keyboards.